src/Winp/Form/ServiceForm.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Diagnostics;
+using System.Drawing; // Added for SystemIcons

```

Comment:
That's a good comment for the PR but can be removed from the codebase 🙂 

```suggestion
using System.Drawing;
```